test/testdrive/sink.td

```diff
@@ -57,9 +57,3 @@ $ kafka-ingest format=avro topic=data schema=${schema} timestamp=43
 > CREATE SINK data_sink FROM data
   INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'data-sink'
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-
-$ kafka-verify format=avro topic=data-sink schema=${schema}
-{"before": null, "after": {"a": 1, "b": 1}}
-{"before": null, "after": {"a": 2, "b": 1}}
-{"before": null, "after": {"a": 3, "b": 1}}
-{"before": null, "after": {"a": 1, "b": 2}}
```

Comment:
probably best to comment this out instead of deleting it so it doesn't get forgotten, but feel free to just file an issue if it saves you a CI run. (I know you're going to work on this next but from personal experience like 1/5 times I get pulled in a different direction and forget unless I leave myself breadcrumbs.)

Reply:
filed https://github.com/MaterializeInc/materialize/issues/2431 won't forget :)